swift-sdk/uicomponents/IterableEmbeddedView.swift

```diff
@@ -91,9 +80,9 @@ public class IterableEmbeddedView:UIView {
     }
 
     /// Primary Button Text
-    public var EMbuttonText: String? = "Placeholding BTN 1" {
+    public var embeddedMessagePrimaryBtnTitle: String? = "Placeholding BTN 1" {
```

Comment:
@evantk91 There's an `EMimage` field immediately above this — should we change that one?

Reply:
There's also an `EMstyle` property further on down